src/workloads/execution/GroupSpillToDisk.yml

```diff
@@ -0,0 +1,126 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: >
+  Runs $group queries which are designed to require spilling to disk.
+Keywords:
+- Loader
+- CrudActor
+- QuiesceActor
+- insert
+- aggregate
+- group
+- spill
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  DocumentCount: &DocumentCount 2_000_000
+  FirstFieldUniqueValCount: &FirstFieldUniqueValCount 20
+  RandomStrLength: &RandomStrLength 60
+  MaxPhases: &MaxPhases 5
+
+Actors:
+# Phase 0: Insert documents into the collection.
+- Name: InsertData
+  Type: Loader
+  Threads: 4
+  Phases:
+    OnlyActiveInPhases:
+      Active: [0]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *Database
+        MultipleThreadsPerCollection: true
+        CollectionCount: 1
+        DocumentCount: *DocumentCount
+        BatchSize: 1000
+        Document:
+          firstField: {^RandomInt: {min: 1, max: *FirstFieldUniqueValCount}}
+          secondField: {^RandomString: {length: *RandomStrLength, alphabet: "0123456789"}}
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *Database
+  Phases:
+    OnlyActiveInPhases:
+      Active: [1]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Threads: 1
+
+# This query has a small number of groups, but each group is a very large set.
+- Name: AddToSetAccumulatorWithLargeSets
+  Type: CrudActor
+  Database: *Database
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Collection: *Collection
+        Operations:
+        - OperationName: aggregate
+          OperationCommand:
+            Pipeline: [{$group: {_id: "$firstField", list: {$addToSet: "$secondField"}}}]
+            Options:
+              AllowDiskUse: true
+
+# Similar to the previous query, but using $push rather than $addToSet.
+- Name: PushAccumulatorConstructingLargeArrays
+  Type: CrudActor
+  Database: *Database
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [3]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Collection: *Collection
+        Operations:
+        - OperationName: aggregate
+          OperationCommand:
+            Pipeline: [{$group: {_id: "$firstField", list: {$push: "$secondField"}}}]
+            Options:
+              AllowDiskUse: true
+
+# This query groups by "secondField", which has many distinct values. Spilling is required since
+# there are many groups, but the state for each individual group is small.
+- Name: SumAccumulatorSpillDueToManyGroups
+  Type: CrudActor
```

Comment:
[question] what is the rationale for supporting aggregates in the CrudActor (i.e. what advantage does this provide [to running aggregates like any other command](https://github.com/mongodb/genny/blob/58901acf3ed1a4dd41f55f41f1d46689ff804e51/src/phases/execution/LookupCommands.yml#L72-L85)) In the description you mention:
`The lack of support for "aggregate" in the CrudActor has led to the use of explain in genny tests in the past`
But it seems that you can get away with not using explain?